baywatch.install

```diff
@@ -29,10 +29,16 @@ function baywatch_import_sdpa_password_policy() {
       $config->delete();
       echo "Password policy with id " . $default_id . " has been deleted.\n";
     }
-    // Import the sdpa password policy.
-    module_load_include('inc', 'tide_core', 'includes/helpers');
-    $config_location = [drupal_get_path('module', 'baywatch') . '/config/install'];
-    _tide_import_single_config('password_policy.password_policy.password_policy_sdpa', $config_location);
+    // Check if the sdpa password policy already exists in the project.
```

Comment:
@MdNadimHossain I don't think this is necessary if the config is instead dropped into the `config/optional` dir.

Reply:
@GROwen I think putting this in the optional folder will work if the project already has this module. But I guess doing the first install in a project which already has this sdpa password policy settings file from somewhere else, will not work.  thoughts?

Reply:
The optional dir is parsed during install as well. Have you tested using this approach? I'm 95% confident that it will work.

Although it is frustrating to throw away something that you've spent time coding up it's good to avoid any customisations where they're not necessary.

Reply:
That is fine, as long as I do it in the right way :) ok I am moving this into the optional directory and removing this check

Reply:
@GROwen Done 👍